cmd/openshift/openshift.go

```diff
@@ -28,6 +30,8 @@ func main() {
 	defer serviceability.BehaviorOnPanic(os.Getenv("OPENSHIFT_ON_PANIC"), version.Get())()
 	defer serviceability.Profile(os.Getenv("OPENSHIFT_PROFILE")).Stop()
 
+	legacy.LegacyInstallAll(legacyscheme.Scheme)
```

Comment:
TODO: remove this.

When is the time coming?

Reply:
> When is the time coming?

not any time soon

Reply:
3.12 is the time, right?